src/components/Dropdowns/SelectionDropdown.tsx

```diff
@@ -0,0 +1,119 @@
+import { Tooltip } from "antd";
+import { ItemType, MenuItemType } from "antd/es/menu/hooks/useItems";
+import React, { ReactElement, useMemo } from "react";
+
+import AccessibleDropdown from "./AccessibleDropdown";
+import DropdownItem, { DropdownItemList } from "./DropdownItem";
+
+type SelectionDropdownProps = {
+  /** Text label to include with the dropdown. If null or undefined, hides the label. */
+  label?: string | null;
+  /** The key of the item that is currently selected. */
+  selected: string;
+  /** An array of ItemType that describes the item properties (`{key, label}`),
+   * or an array of strings. Dropdown items will be presented in the provided array order.
+   *
+   * If a string array is provided, ItemType objects will be
+   * auto-generated with `key` and `label` values set to the string.*/
+  items: ItemType[] | string[];
+  disabled?: boolean;
+  /** The type of button to render for the dropdown. See Antd's button types:
+   * https://ant.design/components/button#components-button-demo-basic */
+  buttonType?: "primary" | "default" | "outlined" | "dashed" | "text" | "link";
```

Comment:
Nit: could this be `ButtonProps["type"]` (or `ButtonType`) as well?

Reply:
Yes, thanks!